src/miral/minimal_window_manager.cpp

```diff
@@ -606,3 +637,12 @@ void miral::MinimalWindowManager::Impl::apply_resize_by(Displacement movement)
         gesture = Gesture::none;
     }
 }
+
+bool miral::MinimalWindowManager::Impl::prevent_focus_stealing(miral::WindowInfo const& info)
+{
+    auto const normal_app_without_parent = (info.depth_layer() == mir_depth_layer_application && !info.parent());
+    auto const decoration_surface = info.type() == mir_window_type_decoration;
```

Comment:
Why are we checking for decorations? Won't that mean that decorations will be raised and focussed regardless of parents and layer? (Which seems wrong)

Reply:
So far, I only see decorations attached to parent windows, so we don't need to check for that. But your comment about the layer is true.